lib/capybara/spec/views/with_view_transition.erb

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
+    <title>with_view_transition</title>
+    <script src="/jquery.js" type="text/javascript" charset="utf-8"></script>
+    <script language="javascript" type="text/javascript">
+      $(document).ready(function(){
+        const input = $("#button");
+        const message = $("#message");
+        input.hide();
+        message.hide();
+
+        $("#root").click(function(event){
+          document.startViewTransition(() => {
+            input.show();
+          });
+        });
+
+        input.click(function(event){
+          message.show();
+        });
```

Comment:
The sequence of events:

1. User clicks link
2. Button slowly fades into view
3. Clicking button reveals a message

I chose this flow because even with animations, the browser is still rendering the content immediately and can be "seen" by Capybara. Issuing a click, however, will not be possible for elements which are animating.